{
    "title" : "Getting \"class CGLNetwork::NetworkAccessDenied: SMTP\" error",
    "uri" : "http://www.salesforce.com/org:organization/articletype:KB_Article/article:kA10d0000004MZ3CAM/language:en_US",
    "printableUri" : "http://www.salesforce.com/org:organization/articletype:KB_Article/article:kA10d0000004MZ3CAM/language:en_US",
    "clickUri" : "https://na61.salesforce.com/kA10d0000004MZ3",
    "uniqueId" : "42.13819$http://www.salesforce.com/org:organization/articletype:KB_Article/article:kA10d0000004MZ3CAM/language:en_US",
    "excerpt" : "Environment ... Cause ... This suggest there is an issue when the Coveo service tries reaching that SMTP server in order error to send out an email. ... Resolution ... Go into Configuration menu > ...",
    "firstSentences" : null,
    "summary" : null,
    "flags" : "HasHtmlVersion",
    "hasHtmlVersion" : true,
    "hasMobileHtmlVersion" : false,
    "score" : 1292,
    "percentScore" : 80.88122,
    "rankingInfo" : null,
    "rating" : 3.0,
    "isTopResult" : false,
    "isRecommendation" : false,
    "titleHighlights" : [ ],
    "firstSentencesHighlights" : [ ],
    "excerptHighlights" : [ ],
    "printableUriHighlights" : [ ],
    "summaryHighlights" : [ ],
    "parentResult" : null,
    "childResults" : [ ],
    "totalNumberOfChildResults" : 0,
    "raw" : {
      "systitle" : "Getting \"class CGLNetwork::NetworkAccessDenied: SMTP\" error",
      "sysurihash" : "YVwmNJwxBFVIxQZF",
      "urihash" : "YVwmNJwxBFVIxQZF",
      "parents" : "<?xml version=\"1.0\" encoding=\"utf-16\"?><parents><parent name=\"Organization\" uri=\"https://na61.salesforce.com/home/home.jsp\" /><parent name=\"KB_Article__ka\" uri=\"http://www.salesforce.com/org:organization/articletype:KB_Article\" /><parent name=\"Getting &quot;class CGLNetwork::NetworkAccessDenied: SMTP&quot; error\" uri=\"https://na61.salesforce.com/kA10d0000004MZ3\" /></parents>",
      "sfdcarticletype" : "How To/Question",
      "sfresolutionc" : "Go into Configuration menu &gt; Alerts &gt; Click on &quot;Send Test Email&quot; and see what happens.<br><br>One possibility: The system (administrators) who configured the system, were passing credentials, and the mail server was configured for anonymous use. When you passed credentials, it failed. <br>If you remove the credentials being passed and allowed it to act as an anonymous user, it will start working again",
      "sysuri" : "http://www.salesforce.com/org:organization/articletype:KB_Article/article:kA10d0000004MZ3CAM/language:en_US",
      "sysprintableuri" : "http://www.salesforce.com/org:organization/articletype:KB_Article/article:kA10d0000004MZ3CAM/language:en_US",
      "sflastpublisheddate" : 1509459579000,
      "systransactionid" : 161139,
      "sfurlname" : "2737",
      "sfsystemmodstamp" : 1513770319000,
      "sfisvisibleinapp" : "true",
      "sfcoveoversionc" : "Coveo Enterprise Search",
      "sysconcepts" : "server ; credentials ; anonymous user ; tries reaching ; Coveo service ; administrators ; Alerts ; Configuration ; Resolution ; Environment",
      "sfkbversionnumber" : "1",
      "concepts" : "server ; credentials ; anonymous user ; tries reaching ; Coveo service ; administrators ; Alerts ; Configuration ; Resolution ; Environment",
      "printableuri" : "http://www.salesforce.com/org:organization/articletype:KB_Article/article:kA10d0000004MZ3CAM/language:en_US",
      "documenttype" : "article",
      "sfkbarticlekavid" : "ka10d0000004SvOAAU",
      "sysindexeddate" : 1518187590000,
      "sfknowledgearticleid" : "kA10d0000004MZ3CAM",
      "sfcreatedbyid" : "00560000002MKBSAA4",
      "sfconnectorc" : "N/A",
      "permanentid" : "d03127023e999c9549d1bcaa5791ffd712afc66a255acfbfd2bdad6f0617",
      "syslanguage" : [ "English" ],
      "transactionid" : 161139,
      "sfisvisibleinprm" : "true",
      "title" : "Getting \"class CGLNetwork::NetworkAccessDenied: SMTP\" error",
      "sfversionnumber" : 1,
      "sfarticlenumber" : "000002737",
      "date" : 1513770319000,
      "objecttype" : "KB Article",
      "allmetadatavalues" : "{\"kav_lastmodifiedby.attribute.type\": [\"User\"], \"coveo_short_id\": [\"kA10d0000004MZ3\"], \"sflastmodifiedbyname\": [\"Karel Mpungi\"], \"kav_coveo_version__c__html_stripped\": [\"Coveo Enterprise Search\"], \"fileextension\": [\"\"], \"sfdcexpandedproduct\": [\"All;All|Coveo Enterprise Search\"], \"kav_createdby.attribute.url\": [\"/services/data/v34.0/sobjects/User/00560000002MKBSAA4\"], \"sflanguage\": [\"en_US\"], \"kav_createdby.attribute.type\": [\"User\"], \"kav_createdby.id\": [\"00560000002MKBSAA4\"], \"kav_language\": [\"en_US\"], \"sfsitecoreversionc\": [\"N/A\"], \"sourcetype\": [\"Salesforce\"], \"title\": [\"Getting \\\"class CGLNetwork::NetworkAccessDenied: SMTP\\\" error\"], \"source\": [\"Salesforce Public Items\"], \"kav_owner.attribute.url\": [\"/services/data/v34.0/sobjects/User/00560000002MKBSAA4\"], \"parents\": [\"<?xml version=\\\"1.0\\\" encoding=\\\"utf-16\\\"?><parents><parent name=\\\"Organization\\\" uri=\\\"https://na61.salesforce.com/home/home.jsp\\\" /><parent name=\\\"KB_Article__ka\\\" uri=\\\"http://www.salesforce.com/org:organization/articletype:KB_Article\\\" /><parent name=\\\"Getting &quot;class CGLNetwork::NetworkAccessDenied: SMTP&quot; error\\\" uri=\\\"https://na61.salesforce.com/kA10d0000004MZ3\\\" /></parents>\"], \"sfisvisibleinprm\": [true], \"kav_title\": [\"Getting \\\"class CGLNetwork::NetworkAccessDenied: SMTP\\\" error\"], \"kav_isvisibleincsp\": [true], \"hashtmlversion\": [true], \"kav_versionnumber\": [1], \"sfcreatedbyname\": [\"Karel Mpungi\"], \"sflastpublisheddate\": [1509459579], \"objecttypelabel\": [\"KB Article\"], \"kav_cause__c__html_stripped\": [\"This suggest there is an issue when the Coveo service tries reaching that SMTP server in order error to send out an email.\"], \"originalhtmlcharset\": [1200], \"coveo_connector_hasbinarydata\": [false], \"kav_lastpublisheddate\": [1509459579], \"sfsystemmodstamp\": [1513770319], \"ka_attribute.type\": [\"KB_Article__ka\"], \"sfdcproduct\": [\"Coveo Enterprise Search\"], \"kav_resolution__c__html_stripped\": [\"Go into Configuration menu > Alerts > Click on \\\"Send Test Email\\\" and see what happens. \\n \\n One possibility: The system (administrators) who configured the system, were passing credentials, and the mail server was configured for anonymous use. When you passed credentials, it failed.\\u00a0 \\n If you remove the credentials being passed and allowed it to act as an anonymous user, it will start working again\"], \"clickableuri\": [\"https://na61.salesforce.com/kA10d0000004MZ3\"], \"sftitle\": [\"Getting \\\"class CGLNetwork::NetworkAccessDenied: SMTP\\\" error\"], \"kav_owner.id\": [\"00560000002MKBSAA4\"], \"sfisvisibleinapp\": [true], \"ka_attribute.url\": [\"/services/data/v34.0/sobjects/KB_Article__ka/kA10d0000004MZ3CAM\"], \"sfdcarticletype\": [\"How To/Question\"], \"kav_cause__c\": [\"This suggest there is an issue when the Coveo service tries reaching that SMTP server in order error to send out an email.\"], \"sfislatestversion\": [true], \"sfcausec\": [\"This suggest there is an issue when the Coveo service tries reaching that SMTP server in order error to send out an email.\"], \"contenttype\": [\"binarydata\"], \"sfurlname\": [\"2737\"], \"ka_id\": [\"kA10d0000004MZ3CAM\"], \"kav_datacategoryselections.datacategoryname\": [\"Coveo_Enterprise_Search;How_To\"], \"coveo_url\": [\"https://na61.salesforce.com/kA10d0000004MZ3\"], \"kav_urlname\": [\"2737\"], \"kav_dc_expanded_product\": [\"All;All|Coveo Enterprise Search\"], \"foldingcollection\": [\"e4a676e14c76d72e541bb016bd206419c27afdafc9b3b8026f1319e60b3c\"], \"body\": [\"<html>   <body>     <div style=\\\"background-color: #fff;border: 2px solid #BBC5CC;border-radius: 4px;float: left;font-family: lato;margin-left: 15px;margin-right: 15px;margin-bottom: 4%;padding: 10px;width: 60%;\\\">       <div style=\\\"font-size: 1.5em;font-weight: 600;Padding-top: 30px;border-bottom: 1px solid #BBC5CC;\\\">         <div style=\\\"border-left: 1px solid #BBC5CC;border-top: 1px solid #BBC5CC;color: #296896;padding: 10px;width: 15%;\\\">           Description         </div>       </div>       <div style=\\\"margin-left: 35px;margin-top: 10px;font-size: 1em;\\\"> This KB explains how to address the &quot;class CGLNetwork::NetworkAccessDenied: SMTP&quot; error:<br>\\u00a0\\n<pre class=\\\"ckeditor_codeblock\\\">\\n|ERROR|2017-10-19|00:00:15|0|An error occured (class CGLNetwork::NetworkAccessDenied: SMTP error on smtp://[SERVER_URL]. (Login denied).) when sending the following alert:</pre>\\n<br>\\u00a0</div>       <div style=\\\"font-size: 1.5em;font-weight: 600;Padding-top: 30px;border-bottom: 1px solid #BBC5CC;\\\">         <div style=\\\"border-left: 1px solid #BBC5CC;border-top: 1px solid #BBC5CC;color: #296896;padding: 10px;width: 15%;\\\">           Environment         </div>       </div>       <br/>       <table>         <tr>           <td style=\\\"margin-left: 35px;margin-top: 10px;font-size: 1em;font-weight: 600;\\\">Product</td>           <td></td>           <td style=\\\"margin-left: 35px;margin-top: 10px;font-size: 1em;font-weight: 600;\\\">Sitecore Version</td>           <td>N/A</td>           <td style=\\\"margin-left: 35px;margin-top: 10px;font-size: 1em;font-weight: 600;\\\">Connector</td>           <td>N/A</td>         </tr>         <tr>           <td style=\\\"margin-left: 35px;margin-top: 10px;font-size: 1em;font-weight: 600;\\\">Published</td>           <td>1509459579</td>           <td style=\\\"margin-left: 35px;margin-top: 10px;font-size: 1em;font-weight: 600;\\\">Version</td>           <td></td>           <td style=\\\"margin-left: 35px;margin-top: 10px;font-size: 1em;font-weight: 600;\\\">Build</td>           <td></td>         </tr>       </table>       <br/>       <div style=\\\"font-size: 1.5em;font-weight: 600;Padding-top: 30px;border-bottom: 1px solid #BBC5CC;\\\">         <div style=\\\"border-left: 1px solid #BBC5CC;border-top: 1px solid #BBC5CC;color: #296896;padding: 10px;width: 15%;\\\">           Cause         </div>       </div>       <div style=\\\"margin-left: 35px;margin-top: 10px;font-size: 1em;\\\"> This suggest there is an issue when the Coveo service tries reaching that SMTP server in order error to send out an email.</div>       <div style=\\\"font-size: 1.5em;font-weight: 600;Padding-top: 30px;border-bottom: 1px solid #BBC5CC;\\\">         <div style=\\\"border-left: 1px solid #BBC5CC;border-top: 1px solid #BBC5CC;color: #296896;padding: 10px;width: 15%;\\\">           Resolution         </div>       </div>       <div style=\\\"margin-left: 35px;margin-top: 10px;font-size: 1em;\\\"> Go into Configuration menu &gt; Alerts &gt; Click on &quot;Send Test Email&quot; and see what happens.<br><br>One possibility: The system (administrators) who configured the system, were passing credentials, and the mail server was configured for anonymous use. When you passed credentials, it failed.\\u00a0<br>If you remove the credentials being passed and allowed it to act as an anonymous user, it will start working again</div>     </div>   </body> </html>\"], \"sfpublishstatus\": [\"Online\"], \"sfcreatedbyid\": [\"00560000002MKBSAA4\"], \"kav_description__c__html_stripped\": [\"This KB explains how to address the \\\"class CGLNetwork::NetworkAccessDenied: SMTP\\\" error: \\n \\u00a0\\n \\n|ERROR|2017-10-19|00:00:15|0|An error occured (class CGLNetwork::NetworkAccessDenied: SMTP error on smtp://[SERVER_URL]. (Login denied).) when sending the following alert: \\n \\n \\u00a0\"], \"kav_islatestversion\": [true], \"kav_knowledgearticleid\": [\"kA10d0000004MZ3CAM\"], \"sfknowledgearticleid\": [\"kA10d0000004MZ3CAM\"], \"sfownername\": [\"Karel Mpungi\"], \"kav_datacategoryselections.attribute.type\": [\"KB_Article__DataCategorySelection;KB_Article__DataCategorySelection\"], \"detectedtitle\": [\"Description\"], \"kav_createddate\": [1509384390], \"language\": [\"English\"], \"coveo_foldingchild\": [\"3ed6d719b150472b64f329577ade121f74424a0a268c8dfaf2132618549b\"], \"sfownerid\": [\"00560000002MKBSAA4\"], \"kav_createdby.coveo_record_size\": [2], \"kav_systemmodstamp\": [1513770319], \"objecttype\": [\"KB Article\"], \"connectortype\": [\"Salesforce2\"], \"kav_coveo_record_size\": [4], \"originaluri\": [\"http://www.salesforce.com/org:organization/articletype:KB_Article/article:kA10d0000004MZ3CAM/language:en_US\"], \"sfisvisibleincsp\": [true], \"convertertype\": [\"TYPE_HTML\"], \"filetype\": [\"SalesforceItem\"], \"kav_owner.name\": [\"Karel Mpungi\"], \"sfcoveoversionc\": [\"Coveo Enterprise Search\"], \"sfdcexpandedarticletype\": [\"All;All|How To/Question\"], \"objecttypelabelplural\": [\"KB Articles\"], \"sfcurrencyisocode\": [\"USD\"], \"sfisdeleted\": [false], \"extractedsize\": [1854], \"size\": [3947], \"kav_publishstatus\": [\"Online\"], \"kav_connector__c\": [\"N/A\"], \"generateexcerpt\": [true], \"sfdescriptionc\": [\"This KB explains how to address the &quot;class CGLNetwork::NetworkAccessDenied: SMTP&quot; error:<br>\\u00a0\\n<pre class=\\\"ckeditor_codeblock\\\">\\n|ERROR|2017-10-19|00:00:15|0|An error occured (class CGLNetwork::NetworkAccessDenied: SMTP error on smtp://[SERVER_URL]. (Login denied).) when sending the following alert:</pre>\\n<br>\\u00a0\"], \"sfarticletype\": [\"KB_Article__kav\"], \"kav_articletype\": [\"KB_Article__kav\"], \"kav_lastmodifieddate\": [1509459579], \"documenttype\": [\"article\"], \"sfkbarticlekavid\": [\"ka10d0000004SvOAAU\"], \"kav_currencyisocode\": [\"USD\"], \"foldingparent\": [\"3ed6d719b150472b64f329577ade121f74424a0a268c8dfaf2132618549b\"], \"mappingtype\": [\"KB_Article\"], \"kav_isvisibleinpkb\": [true], \"kav_sourceid\": [\"5000d00001EYPDyAAP\"], \"sfkbversionnumber\": [1], \"kav_isvisibleinapp\": [true], \"kav_resolution__c\": [\"Go into Configuration menu &gt; Alerts &gt; Click on &quot;Send Test Email&quot; and see what happens.<br><br>One possibility: The system (administrators) who configured the system, were passing credentials, and the mail server was configured for anonymous use. When you passed credentials, it failed.\\u00a0<br>If you remove the credentials being passed and allowed it to act as an anonymous user, it will start working again\"], \"coveo_objecttype\": [\"KB_Article\"], \"printableuri\": [\"http://www.salesforce.com/org:organization/articletype:KB_Article/article:kA10d0000004MZ3CAM/language:en_US\"], \"kav_datacategoryselections.attribute.url\": [\"/services/data/v34.0/sobjects/KB_Article__DataCategorySelection/02o0d000000m5OlAAI;/services/data/v34.0/sobjects/KB_Article__DataCategorySelection/02o0d000000m5OmAAI\"], \"kav_id\": [\"ka10d0000004SvOAAU\"], \"sfresolutionc\": [\"Go into Configuration menu &gt; Alerts &gt; Click on &quot;Send Test Email&quot; and see what happens.<br><br>One possibility: The system (administrators) who configured the system, were passing credentials, and the mail server was configured for anonymous use. When you passed credentials, it failed.\\u00a0<br>If you remove the credentials being passed and allowed it to act as an anonymous user, it will start working again\"], \"foldingchild\": [\"3ed6d719b150472b64f329577ade121f74424a0a268c8dfaf2132618549b\"], \"sfcreateddate\": [1509384390], \"kav_sitecore_version__c\": [\"N/A\"], \"kav_owner.coveo_record_size\": [2], \"kav_coveo_effective_modstamp\": [1513770319], \"kav_summary__html_stripped\": [\"\"], \"kav_dc_article_type\": [\"How To/Question\"], \"sfconnectorc\": [\"N/A\"], \"kav_dc_product\": [\"Coveo Enterprise Search\"], \"ka_coveo_record_size\": [2], \"kav_dc_expanded_article_type\": [\"All;All|How To/Question\"], \"kav_datacategoryselections.coveo_record_size\": [\"2;2\"], \"concepts\": [\"server ; credentials ; anonymous user ; tries reaching ; Coveo service ; administrators ; Alerts ; Configuration ; Resolution ; Environment\"], \"sfkbid\": [\"kA10d0000004MZ3CAM\"], \"kav_firstpublisheddate\": [1509459579], \"coveo_organization_id\": [\"00D3000000007r2\"], \"coveo_foldingcollection\": [\"e4a676e14c76d72e541bb016bd206419c27afdafc9b3b8026f1319e60b3c\"], \"sffirstpublisheddate\": [1509459579], \"sflastmodifieddate\": [1509459579], \"kav_coveo_version__c\": [\"Coveo Enterprise Search\"], \"kav_vendor_version__c__html_stripped\": [\"\"], \"detectedlanguage\": [1], \"kav_isvisibleinprm\": [true], \"kav_owner.attribute.type\": [\"Name\"], \"kav_lastmodifiedby.attribute.url\": [\"/services/data/v34.0/sobjects/User/00560000002MKBSAA4\"], \"sitename\": [\"Community\"], \"kav_createdby.name\": [\"Karel Mpungi\"], \"sfarticlenumber\": [\"000002737\"], \"kav_lastmodifiedby.coveo_record_size\": [2], \"sfisvisibleinpkb\": [true], \"kav_datacategoryselections.id\": [\"02o0d000000m5OlAAI;02o0d000000m5OmAAI\"], \"kav_description__c\": [\"This KB explains how to address the &quot;class CGLNetwork::NetworkAccessDenied: SMTP&quot; error:<br>\\u00a0\\n<pre class=\\\"ckeditor_codeblock\\\">\\n|ERROR|2017-10-19|00:00:15|0|An error occured (class CGLNetwork::NetworkAccessDenied: SMTP error on smtp://[SERVER_URL]. (Login denied).) when sending the following alert:</pre>\\n<br>\\u00a0\"], \"kav_attribute.url\": [\"/services/data/v34.0/sobjects/KB_Article__kav/ka10d0000004SvOAAU\"], \"kav_articlenumber__stripped\": [\"2737\"], \"sfversionnumber\": [1], \"kav_attribute.type\": [\"KB_Article__kav\"], \"kav_validationstatus\": [\"Reviewed\"], \"coveo_foldingparent\": [\"3ed6d719b150472b64f329577ade121f74424a0a268c8dfaf2132618549b\"], \"collection\": [\"default\"], \"sfsourceid\": [\"5000d00001EYPDyAAP\"], \"date\": [1513770319], \"kav_lastmodifiedby.id\": [\"00560000002MKBSAA4\"], \"kav_datacategoryselections.datacategorygroupname\": [\"Product;Article_Type\"], \"conversionstate\": [0], \"sfvalidationstatus\": [\"Reviewed\"], \"kav_isdeleted\": [false], \"sflastmodifiedbyid\": [\"00560000002MKBSAA4\"], \"kav_lastmodifiedby.name\": [\"Karel Mpungi\"], \"kav_articlenumber\": [\"000002737\"], \"permanentid\": [\"d03127023e999c9549d1bcaa5791ffd712afc66a255acfbfd2bdad6f0617\"]}",
      "sflastmodifiedbyid" : "00560000002MKBSAA4",
      "syssfownerid" : "00560000002MKBSAA4",
      "sfsitecoreversionc" : "N/A",
      "sourcetype" : "Salesforce",
      "sysconnectortype" : "Salesforce2",
      "sftitle" : "Getting \"class CGLNetwork::NetworkAccessDenied: SMTP\" error",
      "rowid" : 5493174,
      "syssfcreatedbyid" : "00560000002MKBSAA4",
      "sfdescriptionc" : "This KB explains how to address the &quot;class CGLNetwork::NetworkAccessDenied: SMTP&quot; error:<br> \n<pre class=\"ckeditor_codeblock\">\n|ERROR|2017-10-19|00:00:15|0|An error occured (class CGLNetwork::NetworkAccessDenied: SMTP error on smtp://[SERVER_URL]. (Login denied).) when sending the following alert:</pre>\n<br> ",
      "size" : 3947,
      "sfislatestversion" : "true",
      "sysdocumenttype" : "article",
      "sfcreateddate" : 1509384390000,
      "sfsourceid" : "5000d00001EYPDyAAP",
      "clickableuri" : "https://na61.salesforce.com/kA10d0000004MZ3",
      "syssource" : "Salesforce Public Items",
      "sfownerid" : "00560000002MKBSAA4",
      "orderingid" : 212384990660371979,
      "sfpublishstatus" : "Online",
      "syssize" : 3947,
      "sysdate" : 1513770319000,
      "sflanguage" : "en_US",
      "sysparents" : "<?xml version=\"1.0\" encoding=\"utf-16\"?><parents><parent name=\"Organization\" uri=\"https://na61.salesforce.com/home/home.jsp\" /><parent name=\"KB_Article__ka\" uri=\"http://www.salesforce.com/org:organization/articletype:KB_Article\" /><parent name=\"Getting &quot;class CGLNetwork::NetworkAccessDenied: SMTP&quot; error\" uri=\"https://na61.salesforce.com/kA10d0000004MZ3\" /></parents>",
      "sfdcexpandedarticletype" : "All;All|How To/Question",
      "sfcurrencyisocode" : "USD",
      "sfkbid" : "kA10d0000004MZ3CAM",
      "sfcausec" : "This suggest there is an issue when the Coveo service tries reaching that SMTP server in order error to send out an email.",
      "sfdcexpandedproduct" : "All;All|Coveo Enterprise Search",
      "syssfcreateddate" : 1509384390000,
      "sflastmodifieddate" : 1509459579000,
      "sfisvisibleinpkb" : "true",
      "source" : "Salesforce Public Items",
      "sfisdeleted" : "false",
      "sfisvisibleincsp" : "true",
      "sfvalidationstatus" : "Reviewed",
      "sflastmodifiedbyname" : "Karel Mpungi",
      "collection" : "default",
      "sfdcproduct" : [ "Coveo Enterprise Search" ],
      "syssourcetype" : "Salesforce",
      "sfarticletype" : "KB_Article__kav",
      "indexeddate" : 1518187590000,
      "connectortype" : "Salesforce2",
      "filetype" : "SalesforceItem",
      "sfcreatedbyname" : "Karel Mpungi",
      "sysclickableuri" : "https://na61.salesforce.com/kA10d0000004MZ3",
      "sysfiletype" : "SalesforceItem",
      "language" : [ "English" ],
      "sffirstpublisheddate" : 1509459579000,
      "sfownername" : "Karel Mpungi",
      "sitename" : "Community",
      "sysrowid" : 5493174,
      "uri" : "http://www.salesforce.com/org:organization/articletype:KB_Article/article:kA10d0000004MZ3CAM/language:en_US",
      "syscollection" : "default"
    },
    "Title" : "Getting \"class CGLNetwork::NetworkAccessDenied: SMTP\" error",
    "Uri" : "http://www.salesforce.com/org:organization/articletype:KB_Article/article:kA10d0000004MZ3CAM/language:en_US",
    "PrintableUri" : "http://www.salesforce.com/org:organization/articletype:KB_Article/article:kA10d0000004MZ3CAM/language:en_US",
    "ClickUri" : "https://na61.salesforce.com/kA10d0000004MZ3",
    "UniqueId" : "42.13819$http://www.salesforce.com/org:organization/articletype:KB_Article/article:kA10d0000004MZ3CAM/language:en_US",
    "Excerpt" : "Environment ... Cause ... This suggest there is an issue when the Coveo service tries reaching that SMTP server in order error to send out an email. ... Resolution ... Go into Configuration menu > ...",
    "FirstSentences" : null
  }